core/src/main/java/kafka/log/remote/RemoteLogManager.java

```diff
@@ -285,19 +288,15 @@ public RemoteStorageManager storageManager() {
         return remoteLogStorageManager;
     }
 
+    public MetadataCache metadataCache() {
+        return metadataCache;
```

Comment:
Why do we need this? For test only? I think the test doesn't have to use it. See my comment below.